docs/setup-cluster/on-prem/options/wsl.rst

```diff
@@ -52,379 +52,138 @@ Edit the configuration file to enable ``systemd`` within your WSL distribution.
 
-Pull the PostgreSQL Image
-=========================
+A configuration file is needed to set important values in the master, such as where to save model
+checkpoints. For information about how to create a configuration file, see
+:ref:`cluster-configuration`. There are also sample configuration files available.
 
-Pull the official Docker image for PostgreSQL. We recommend using the version listed below.
-
-.. code::
-
-   docker pull postgres:10
-
-This image is not provided by Determined AI. For more information, visit its `Docker Hub page
-<https://hub.docker.com/_/postgres>`_.
+.. note::
 
-.. _docker_desktop_determined:
+   ``det deploy`` will use a default configuration file if you don't provide one. It also
+   transparently manages PostgreSQL along with the master, so the configuration options related to
+   those services do not need to be set.
 
-Pull the Determined AI Image
+Deploy a Single-Node Cluster
 ============================
 
-Pull the Docker image for the master or agent on each machine where these services will run. There
-is a single master container running in a Determined cluster and typically there is one agent
-container running on a given machine. A single machine can host both the master container and an
-agent container.
-
-To run the commands below, replace ``VERSION`` with a valid Determined version, such as the current
-version, |version|:
+For local development or small clusters (such as a GPU workstation), you may wish to install both a
+master and an agent on the same node. To do this, run one of the following commands:
 
 .. code::
 
-   docker pull determinedai/determined-master:VERSION
-   docker pull determinedai/determined-agent:VERSION
-
-.. _docker_desktop_start_cluster:
+   # If the machine has GPUs:
+   det deploy local cluster-up
 
-Start the Cluster
-=================
+   # If the machine doesn't have GPUs:
+   det deploy local cluster-up --no-gpu
 
-The cluster can now be started, first by starting the database, then by launching the Determined
-master and agent containers.
+This will start a master and an agent on that machine. To verify that the master is running,
+navigate to ``http://localhost:8080`` in a browser, which should bring up the Determined
+WebUI.
 
-.. _docker_desktop_start_postgresql:
-
-Start the PostgreSQL Container
-==============================
-
-To start the PostgreSQL container, use the following command. Replace ``<DB password>`` with the
-password you would like to use for the database:
+To open the WebUI from WSL:
 
 .. code::
 
-   docker run \
-       --name determined-db \
-       -p 5432:5432 \
-       -v determined_db:/var/lib/postgresql/data \
-       -e POSTGRES_DB=determined \
-       -e POSTGRES_PASSWORD=<DB password> \
-       postgres:10
-
-.. _docker_desktop_get_wsl_ip:
+   explorer.exe http://localhost:8080
 
-Obtain the WSL IP Address
-=========================
+The default username for the WebUI is ``admin`` and no password.
```

Comment:
is it ''admin'' or ''determined''? i guess both work.

i've been using some variation of the following:

-  Go to ``http://localhost:8080/``.
-  Accept the default username of ``determined`` and leave the password empty.
-  Click **Sign In**.

Reply:
Both work. Think we should mention both?